Blog: Bitcoin Grants 6th Wave (#294)
OpenSats/website: data/blog/bitcoin-grants-6th-wave.mdx

---
title: "Sixth Wave of Bitcoin Grants"
date: '2024-07-22'
tags: ['OpenSats', 'grants', 'bitcoin']
draft: false
authors: ['arvin', 'dergigi']
images: ['/static/images/blog/TODO.jpg']
summary: "OpenSats is funding 8 more open-source projects in the bitcoin ecosystem."
---

We are excited to announce the sixth wave of Bitcoin grants, reaffirming our commitment to fostering innovation and development within the open-source community. 
An important part of our mission is to provide essential funding to developers and projects that advance the Bitcoin ecosystem.

> In this latest round, we are proud to support eight remarkable projects that exemplify the spirit of open-source and technological advancement. In no particular order, these projects are:

Reviewer: ```suggestion
In this latest round, we are proud to support eight remarkable projects that exemplify the spirit of our mission. In no particular order, these projects are:
```

> -   [Bitcoin Jungle](#bitcoin-jungle)

Reviewer: I'd suggest we move Bitcoin Jungle to the end of the list. We usually lead with the tech, and end on education.

Author: Done 🖖

-   [secp256k1.swift](#secp256k1.swift)
-   [Blitz Wallet](#blitz-wallet)
-   [NLightning](#nlightning)
-   [Bitcoin Fees](#bitcoin-fees)
-   [eNuts](#enuts)
-   [Peer Storage Backups for Lightning Channels](#peer-storage-backups-for-lightning-channels)
-   [Gonuts](#gonuts)

> Made possible by the generosity of our donors, these grants enable contributors and projects to continue their critical work and drive the future of Bitcoin forward. 

Reviewer: ```suggestion
Made possible by the generosity of our donors, these grants enable the various contributors to continue their critical work into the future.
```

features into their own applications.

The grant will support several key developments for Gonuts by implementing new protocol additions such as Spending Conditions and Pay to Public Key (as outlined in NUTs [10](https://github.com/cashubtc/nuts/blob/main/10.md) and [11](https://github.com/cashubtc/nuts/blob/main/11.md)), enabling "Multinut" payments to allow users to pay a Lightning invoice from multiple mints, and improving mint reliability by adding support for [SQLite](https://www.sqlite.org/).

Additionally, the grant will facilitate the creation of a [Docker container](https://www.docker.com/resources/what-container/) for easier mint deployment and extend support to other 
Lightning backends beyond LND. These enhancements will make Gonuts more robust, versatile, and user-friendly, fostering broader adoption and innovation within the Cashu ecosystem.

Repository: [elnosh/gonu](https://github.com/elnosh/gonuts)

License: MIT

---

With this 6th wave of grants, much like our prior waves, we evaluated applications based on carefully defined [criteria](/apply#criteria) and did our best to select only the most impactful 

> and promising proposals for the Bitcoin ecosystem. If you are a developer working on a project that is aligned with [our mission](https://opensats.org/mission), don't hesitate to [apply for funding](/apply).

Reviewer: ```suggestion
and promising proposals for the Bitcoin ecosystem. If you are a developer working on a project that is aligned with [our mission](/mission), don't hesitate to [apply for funding](/apply).
```
